### DISCUSSION TITLE
Excess Warbonds 'Issue'

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#1447] [Ideas]
I have seen discussions around the 'issues' of having excess warbonds at the end of the war after every item is minted out, the issue is that by the end of the war there will be people with leftover warbonds with no chance of using them for anything.

One idea could be to introduce a tiered decoration reward with enough supply for everyone:

It's one item, let's say a sword, with 20k supply which unlocks on the last day of the war or whenever every item gets minted out (including the hidden rewards, which i assume some people will miss out on). It has different levels of rarity, if you have 100 warbonds leftover, you get a wooden sword, if you have 1k wb leftover you get a stone sword..etc.etc. if you have 25k warbonds you get a diamond sword. This item would be DECORATIVE, no utility, therefore it will not affect the economy and gives a chance for everyone to get an item in return for their warbonds which would otherwise be sitting there doing nothing.

**WARNING: NOT AN ARTIST AND NEVER WILL BE**

![image](https://user-images.githubusercontent.com/108110065/192419129-8fec8520-511f-4fd5-893f-b7ef30d41658.png)

Obviously people would be able to only select 1 of the items, not all of them.

What do you think? Obviously it does not have to be a sword (there already is a goblin sword..) it can be something else. Something like this would eliminate excess warbonds and give the people holding 'useless' warbonds (Useless if all nfts are minted out) a way to burn them and get something decorative in return.

Peace and Love, Go Goblins

**Comment by S2LKR** [comment]
Seem great idea! Decorative nft as to commemorate community member's contribution (besides warrior badge for us regular players). 

Hope this will be less burden for dev to invest time on, compared to more "complicated"  nft with boost or convert to xp and focus on launching LE. At the same time, opportunity for war participants to convert excess WB for in-game nft (without boost). As you noted, perhaps doesn't have to be a sword, but some kind of decorative nft would be great.
